pages/services/kafka/2.6.0-2.2.1/limitations/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Known limitations for the DC/OS Apache Kafka service
+title: Limitations
+menuWeight: 100
+model: /services/kafka/data.yml
+render: mustache
+---
+
+#include /services/include/limitations.tmpl
+#include /services/include/limitations-zones.tmpl
+#include /services/include/limitations-regions.tmpl
+
+## Log Retention Bytes
+
+The `disk` configuration value is denominated in MB. We recommend that you set the configuration value `log_retention_bytes` to a value smaller than the indicated `disk` configuration. See the [Configuration](/services/kafka/2.5.0-2.1.0/configuration/) section for instructions on customizing these values.
+
+## Security
+
+### {{ model.techShortName }} CLI
+
+When any security functions are enabled, the {{ model.techShortName }} service CLI sub-command `topic` will not function. While the service CLI convenience functions will not work, the tooling bundled with [{{ model.techName }}](https://cwiki.apache.org/confluence/display/KAFKA/System+Tools) and other tools that support the enabled security modes will work.
```

Comment:
Is this still true?

Reply:
Yes it's true for some sub-commands of topic like: offset, producer_test, etc. Please check https://jira.mesosphere.com/browse/DCOS-20517 for more details.